internal/pkg/manifest/lb_web_svc_test.go

```diff
@@ -154,6 +154,7 @@ func TestNewLoadBalancedWebService_UnmarshalYaml(t *testing.T) {
 					Interval:           durationp(78 * time.Second),
 					Timeout:            durationp(9 * time.Second),
 				},
+				HealthCheckPath: nil,
```

Comment:
can we update the name of this test case to reflect this change? for example:
"should use custom healthcheck configuration when provided and set default path to nil"

Reply:
done!